api/meta_protocol_proxy/v1alpha/meta_protocol_proxy.proto

```diff
@@ -151,3 +154,10 @@ message Tracing {
   envoy.config.trace.v3.Tracing.Http provider = 9;
 }
 
+message ProtocolConfig {
```

Comment:
I'm thinking that we name it ApplicationProtocol and put everything configuration about the application protocol in it.